### ISSUE TITLE
OpenXmlValidator Throws Incorrect Validation Error for CategoryAxis

### ISSUE THREAD

**Issue by ScottHaney** [OfficeDev/Open-XML-SDK#689]
**Description**

When validating a `CategoryAxis` with an `AxisPosition` child the `OpenXmlValidator` creates a validation error even though the xml is valid according the office xml specification (CT_CatAx has a child EG_AxShared which has an element CT_AxPos on page 4080 of part I of the specification).

**Information**

- .NET Target: .NET Framework
- DocumentFormat.OpenXml Version: 2.10

**Repro**

```csharp
var categoryAxis = new DocumentFormat.OpenXml.Drawing.Charts.CategoryAxis();
categoryAxis.AppendChild(new DocumentFormat.OpenXml.Drawing.Charts.AxisPosition() { Val = new EnumValue<DocumentFormat.OpenXml.Drawing.Charts.AxisPositionValues>(DocumentFormat.OpenXml.Drawing.Charts.AxisPositionValues.Bottom) });

var validator = new Validation.OpenXmlValidator();
var errors = validator.Validate(categoryAxis);
``` 

**Observed**

The `errors` variable has a single entry with a description of `The element has unexpected child element 'http://schemas.openxmlformats.org/drawingml/2006/chart:axPos.'...`

**Expected**

According to `CT_CatAx` in part I of the specification (page 4080) there must be child elements `axId`, `scaling`, `axPos`, and `crossAx`. The sample above only has a `axPos` child so I would expect to see three validation errors due to the other three required elements being missing. I would not expect to see a validation error saying that `axPos` is not a valid child since the specification says that it is.

**Comment by twsouthwick** [comment]
@tomjebo and I were talking about a similar error this morning. He's looking into it. We may have some out of date schema info it seems.

**Comment by twsouthwick** [comment]
@tomjebo Thoughts on this?

**Comment by twsouthwick** [comment]
@ScottHaney I can't remember if this was addressed with the changes in 2.11. Are you still seeing the issue?
